### PR TITLE
Wrap totalwealth value with USD.parse [ISSUE #7]

### PR DESCRIPTION
**Pull request by JohnLins**
## Overview

In this PR, I used USD.parse() to allow OurTable to properly sort the totalWealth column of the leaderboard table.

## Screenshots 
image
image
image

## Feedback Request

Please give suggestions on the following:
- Please interact with my branch on dokku and provide your feedback if needed

## Validation 
Interact with the change on my branch here:
https://cow-john.dokku-11.cs.ucsb.edu

## Tests

- [x] Backend Unit tests (`mvn test`) pass
- [x] Backend Test coverage (`mvn test jacoco:report`) 100%
- [x] Backend Mutation tests (`mvn test pitest:mutationCoverage`) 100% 
- [x] Frontend Unit tests (`npm test`) pass
- [x] Frontend Test coverage (`npm run coverage`) 100%
- [x] Frontend Mutation tests (`npx stryker run`) 100% 
- [x] Frontend Linting (`npx eslint --fix src`) 

## Linked Issues

Closes #0

### CODE REVIEW

File: frontend/src/main/components/Leaderboard/LeaderboardTable.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
 import OurTable from "main/components/OurTable";
 import { Link } from "react-router-dom";
 
+
+
```
Comment (smmzhu) [inline frontend/src/main/components/Leaderboard/LeaderboardTable.js:5]: remove the whitespace

File: .java-version [hunk 2]
```diff
@@ -1 +1 @@
-17
```
Comment (smmzhu) [inline .java-version:1]: why the change?

**Review by smmzhu — CHANGES_REQUESTED** [review]
Fix comments

File: frontend/src/main/components/Leaderboard/LeaderboardTable.js [hunk 3]
```diff
@@ -18,12 +18,14 @@ export default function LeaderboardTable({ leaderboardUsers }) {
         },
         {
             Header: "Total Wealth",
+           
             id: "totalWealth",
             accessor: (row, _rowIndex) => {
-                return USD.format(row.totalWealth);
+                return row.totalWealth;
             },
+
```
Comment (smmzhu) [inline frontend/src/main/components/Leaderboard/LeaderboardTable.js:26]: remove space

File: frontend/src/main/components/Leaderboard/LeaderboardTable.js [hunk 4]
```diff
@@ -18,12 +18,14 @@ export default function LeaderboardTable({ leaderboardUsers }) {
         },
         {
             Header: "Total Wealth",
+
```
Comment (smmzhu) [inline frontend/src/main/components/Leaderboard/LeaderboardTable.js:21]: remove space

**Review by smmzhu — APPROVED** [review]
LGTM

**Review by pconrad — APPROVED** [review]
LGTM.  I'd prefer if there were a test, but I've factored that out into a separate issue, #39

**Pull request merged**